src/context.rs

```diff
@@ -21,6 +21,7 @@ pub mod global {
     use {Secp256k1, All};
 
     /// Proxy struct for global `SECP256K1` context
+    #[derive(Debug, Copy, Clone)]
```

Comment:
hmmm, on one hand having a `__private` probably means you don't want it to be Copy/Clone, so I might even open an issue on clippy about it warning here
on the other, the way the implementation here works it should be fine to even make this a `pub struct GlobalContext;` where users can generate their own instances (not sure why they'll want but whatever)

So overall not sure if this should be done or not